components/tab/TabGroup.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class TabGroup extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
```

Comment:
children should be required

Reply:
Argh I keep forgetting to think about this. In that case children is not required either in `ButtonGroup.js`, I might fix that (in other PR)

 :ok: by @InstaK on 29/11/2017 16:34:47